Cogs.Console/Cogs.Console.csproj

```diff
@@ -12,6 +12,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.1.1" />
     <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
+    <PackageReference Include="Newtonsoft.Json.Schema" Version="3.0.1" />
```

Comment:
We can't use the Newtonsoft.Json.Schema as it is GPL, and this project is MIT licensed. Lets remove it from the test project and think of another solution.